pyserini/index/pyutils.py

```diff
@@ -336,3 +336,22 @@ def convert_collection_docid_to_internal_docid(self, docid: str) -> int:
             The Lucene internal ``docid`` corresponding to the external collection ``docid``.
         """
         return self.object.convertDocidToLuceneDocid(self.reader, docid)
+
+    def stats(self) -> dict:
+        """Returns dictionary with index statistics.
+
+        Returns
+        -------
+        dict
+            Index statistics as a dictionary of statistic's name to statistic.
```

Comment:
would be nice to just list the statistics we provide? so user doesn't need to go hunting in the Java code?